### ISSUE TITLE
ID oyster diploid/triploid families

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1689]
Some pics from PW...

<img width="1115" alt="image" src="https://github.com/RobertsLab/resources/assets/4442039/23fd5590-e0f8-46d1-945e-761665757879">

There were 4 bags in one upweller. we color coded the four bags...

<img width="895" alt="image" src="https://github.com/RobertsLab/resources/assets/4442039/c14f1826-119b-44b5-9df6-c419d148c88f">

<img width="815" alt="image" src="https://github.com/RobertsLab/resources/assets/4442039/9ca2e65e-8a80-45f8-98de-eab0e7f60ba3">

<img width="986" alt="image" src="https://github.com/RobertsLab/resources/assets/4442039/a7655183-b9b8-4654-aae5-e97f5262259b">

<img width="967" alt="image" src="https://github.com/RobertsLab/resources/assets/4442039/a5a9965d-823a-4d7f-b2b8-360407152773">

<img width="795" alt="image" src="https://github.com/RobertsLab/resources/assets/4442039/b4b259c4-7de4-4765-8c85-ee3ed4c2a9db">

@AHuffmyer might have more useful info to add

Blues were clearly biggest and had the largest numbers

**Comment by mattgeorgephd** [comment]
I took a few photos of the upwellers when I placed the bags in there. Here is a [presentation with the pictures](https://www.dropbox.com/scl/fi/bvyp96koxqcdxuwadufbj/diptripfamilypictures.pptx?rlkey=oktbyx35kghcca8qcxcxkiy0o&dl=0) and my best guesses as to which families are in which. As memory also serves, there were not very many F5 3n, but they were big!

**Comment by sr320** [comment]
I did locate four more bags and will upload details soon

**Comment by AHuffmyer** [comment]
Do we have updates to this list or do unknowns remain unknown?

**Comment by sr320** [comment]
No, we really need to coordinate with Pacific hybrid to determinative if
they can id with cytometer (dip vs trip)

**Comment by AHuffmyer** [comment]
Is that something we can do or would we need to send them animals?

**Comment by mattgeorgephd** [comment]
Benoit at Quilcene Hatchery also has a ploidy analyzer

**Comment by sr320** [comment]
Alexa (PH) has agreed to do this. The next step is for me to somehow get the oysters to her in Manchester.

**Comment by AHuffmyer** [comment]
I can take oysters to Manchester next week - I could pick up on a Point Whitney day and deliver to Manchester.

**Comment by AHuffmyer** [comment]
Delivered to Manchester on 5/7; Alexa analyzing on 4/8

**Comment by AHuffmyer** [comment]
Results from analyzed cohorts here: https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/Oyster-ploidy-results/

Need to confirm other families not detailed in this post

**Comment by sr320** [comment]
before

Orange = ?
Pink = ?
Blue = ?
Yellow = F14 (2n)
Dog [Putnam hobo] = F5 (2n)?
Cat [very big, few] = F5 (3n)
Bird [Tank 3 hobo] = F13 (either 2n or 3n)
Toad = F14 (3n)


AFTER analyzes

Orange = ? triploid
Pink = ?
Blue = ? Tetraploid 
Yellow = F14 (2n) diploid
Dog [Putnam hobo] = F5 (2n)? triploid
Cat [very big, few] = F5 (3n)
Bird [Tank 3 hobo] = F13 (either 2n or 3n) triploid 
Toad = F14 (3n)

**Comment by AHuffmyer** [comment]
I believe we have done all the identification we can do up to this point, correct? I'll close if so.

**Comment by AHuffmyer** [comment]
@sr320 the Pink family was used in the lifestage carryover project and it would be good to ID if diploid or triploid. I have samples in ethanol for DNA. How would you like to proceed with identifying these? One option would be a microsat method like this one: https://pmc.ncbi.nlm.nih.gov/articles/PMC4093491/
